scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -210,6 +216,19 @@ def discover(self):
                     logger.debug(f'Quarantine file {quarantine_file} is empty')
             self.quarantine = Quarantine(ql)
 
+        #handle extra scripts
+        sl = self.options.scripting_list
+        if sl:
+            for scripting_file in sl:
```

Comment:
can this code also be a part of the new `Scripting` class ?

Reply:
moved